integration/datagenerator/rollup.go

```diff
@@ -1,17 +1,24 @@
 package datagenerator
 
 import (
+	gethcommon "github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
 
 	"github.com/ten-protocol/go-ten/go/common"
 )
 
 // RandomRollup - block is needed in order to pass the smart contract check
 // when submitting cross chain messages.
-func RandomRollup(_ *types.Header) common.ExtRollup {
+// TODO @will this wont work now since we verify the data on the management contract
+func RandomRollup(block *types.Header) common.ExtRollup {
```

Comment:
Isn't this for in mem? We dont have verification on the l1 there so this might pass

Reply:
TestManagementContract

Reply:
which is why they're commented out for now in `smartcontract_tests.go`